pkg/trace/api/api_test.go

```diff
@@ -777,22 +777,6 @@ func TestHandleTraces(t *testing.T) {
 	assert.Equal("C#|go|java|python|ruby", receiver.Languages())
 }
 
-// chunkedReader is a reader which forces partial reads, this is required
```

Comment:
I know this code is going away, but what???
Not reading a full body isn't a server bug...
I'm guessing this was because the old scheme required making sure the full payload was consumed and in-memory.

Reply:
I think this was only used in tests to try and trigger some (unfortunately undescribed) bad behavior. The related test that used this is gone because the rate limiter is gone, so we don't need it. I can't figure out what exactly it was trying to test since, as you say, not reading a full body isn't a bug.